FaunaDB.Client/Query/Expr.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using Newtonsoft.Json;
 using System;
+using System.Reflection;
```

Comment:
*blinks* was this an existing missing dep?

Reply:
no, this is necessary as it brings some extension methods for old frameworks